ngraph/core/src/op/loop.cpp

```diff
@@ -318,94 +318,14 @@ void op::v5::Loop::validate_and_infer_types()
 std::shared_ptr<Node> op::v5::Loop::clone_with_new_inputs(const OutputVector& new_args) const
 {
     NGRAPH_OP_SCOPE(v5_Loop_clone_with_new_inputs);
-    // WA: input description with index 0 or 1 means that Loop consructor will duplicate it in
-    // the inputs.
-    // When using visit_attributes() no duplication occurs, input_offset shall be decremented.
-    size_t input_offset = 2;
-    for (const auto& in_desc : m_input_descriptions)
-    {
-        if (in_desc->m_input_index == 0 || in_desc->m_input_index == 1)
-        {
-            input_offset--;
-        }
-    }
-    // input_offset < 0 means that there are several duplications of external_port_id
-    // (the same ext_port_id is connected to several Parameters in the port map) in input_desc,
-    // this can lead to wrong or undefined behavior, so throw exception here. Ticket: 47302
-    NODE_VALIDATION_CHECK(this, input_offset >= 0, "External port id 0 or 1 is duplicated.");
-    // 0 - trip_count, 1 - execution condition, these inputs are not connected to the body
-    // params
-    OutputVector body_params_args(new_args.begin() + input_offset, new_args.end());
-    auto op = make_shared<op::v5::Loop>(new_args[0], new_args[1]);
-    for (int idx = 2; idx < new_args.size(); ++idx)
-    {
-        op->set_argument(idx, new_args[idx]);
-    }
+    auto op = make_shared<op::v5::Loop>();
```

Comment:
please add `check_new_args_count(this, new_args);` call before this line

Reply:
Nice catch, thanks!

Reply:
Fixed